sdks/java/core/src/test/java/org/apache/beam/sdk/util/ReduceFnRunnerTest.java

```diff
@@ -604,8 +630,8 @@ public void testPaneInfoFinalAndOnTime() throws Exception {
     ReduceFnTester<Integer, Iterable<Integer>, IntervalWindow> tester = ReduceFnTester.nonCombining(
         WindowingStrategy.of(FixedWindows.of(Duration.millis(10)))
             .withTrigger(
-                Repeatedly.<IntervalWindow>forever(AfterPane.<IntervalWindow>elementCountAtLeast(2))
-                    .orFinally(AfterWatermark.<IntervalWindow>pastEndOfWindow()))
+                Repeatedly.<IntervalWindow>forever(AfterPane.elementCountAtLeast(2))
```

Comment:
These weren't reported as unused parameter warnings. I suspect because the methods in question still accept a  generic (`public static <W extends BoundedWindow> Repeatedly forever(Trigger repeated)`) while the ones that I fixed no longer accept one.